opm/parser/eclipse/EclipseState/Schedule/Action/Condition.hpp

```diff
@@ -0,0 +1,70 @@
+/*
+  Copyright 2019 Equinor ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef ACTIONX_CONDITION_HPP
+#define ACTIONX_CONDITION_HPP
+
+#include <string>
+
+namespace Opm {
+namespace Action {
+
+class Condition {
+public:
+
+enum class Logical {
+    AND,
+    OR,
+    END
+};
+
+
+enum class Operator {
```

Comment:
The term Operator makes me expect addition, subtraction etc., although the comparison *operators* are operators too. What about using Comparator?

Reply:
`Comparator` was a good suggestion :+1: